hummingbot/core/gateway/gateway_http_client.py

```diff
@@ -100,29 +102,31 @@ def log_error_codes(self, resp: Dict[str, Any]):
             elif error_code == GatewayError.LoadWallet.value:
-                self.logger().info("Gateway failed to load your wallet. Try running 'gateway connect' with the correct wallet settings.")
+                self.logger().network("Gateway failed to load your wallet. Try running 'gateway connect' with the correct wallet settings.")
             elif error_code == GatewayError.TokenNotSupported.value:
-                self.logger().info("Gateway tried to use an unsupported token.")
+                self.logger().network("Gateway tried to use an unsupported token.")
             elif error_code == GatewayError.TradeFailed.value:
-                self.logger().info("The trade on gateway has failed.")
+                self.logger().network("The trade on gateway has failed.")
+            elif error_code == GatewayError.PriceFailed.value:
+                self.logger().network("The price query on gateway has failed.")
             elif error_code == GatewayError.ServiceUnitialized.value:
-                self.logger().info("Some values was uninitialized. Please contact dev@hummingbot.io ")
+                self.logger().network("Some values was uninitialized. Please contact dev@hummingbot.io ")
             elif error_code == GatewayError.SwapPriceExceedsLimitPrice.value:
-                self.logger().info("The swap price is greater than your limit buy price. The market may be too volatile or your slippage rate is too low. Try adjusting the strategy's allowed slippage rate.")
+                self.logger().network("The swap price is greater than your limit buy price. The market may be too volatile or your slippage rate is too low. Try adjusting the strategy's allowed slippage rate.")
             elif error_code == GatewayError.SwapPriceLowerThanLimitPrice.value:
-                self.logger().info("The swap price is lower than your limit sell price. The market may be too volatile or your slippage rate is too low. Try adjusting the strategy's allowed slippage rate.")
+                self.logger().network("The swap price is lower than your limit sell price. The market may be too volatile or your slippage rate is too low. Try adjusting the strategy's allowed slippage rate.")
             elif error_code == GatewayError.UnknownChainError.value:
-                self.logger().info("An unknown chain error has occurred on gateway. Make sure your gateway settings are correct.")
+                self.logger().network("An unknown chain error has occurred on gateway. Make sure your gateway settings are correct.")
             elif error_code == GatewayError.UnknownError.value:
-                self.logger().info("An unknown error has occurred on gateway. Please send your logs to dev@hummingbot.io")
+                self.logger().network("An unknown error has occurred on gateway. Please send your logs to dev@hummingbot.io")
```

Comment:
@james-hummingbot 
These messages will no longer show on the log panel.
Is that your intention?